lib/migration/src/Obelisk/Migration.hs

```diff
@@ -95,7 +95,7 @@ readGraph root name = doesDirectoryExist root >>= \case
       let f = root </> (T.unpack $ v1 <> "-" <> v2) </> T.unpack graph
       doesFileExist f >>= \case
         True -> do
-          c <- readFile f >>= pure . parseEdgeMeta
+          c <- readFileStrict f >>= pure . parseEdgeMeta
```

Comment:
but if we use T.readFile - what to do with parseEdgeMeta - just drop it? At the moment it is just a T.pack anyway, but I assume this is intended to do more at some point? Should I preserve the `Action` class and change it to:

```haskell
class Action action where
  parseEdgeMeta :: Text -> action

instance Action Text where
  parseEdgeMeta = id
```

Reply:
@srid ?

Reply:
@eskimor Yes, preserve but change it to use `Text` and `id` like you've written above.

Reply:
awesome!